balancer-js/src/modules/liquidity-managment/migrations.ts

```diff
@@ -0,0 +1,118 @@
+import { Findable, Pool, PoolAttribute } from '@/types';
+import { JsonRpcProvider } from '@ethersproject/providers';
+import { ERC20 } from '../contracts/implementations/ERC20';
+import { SubgraphLiquidityGauge } from '../subgraph/subgraph';
+import { migrationBuilder } from './migrations/builder';
+import { buildMigrationPool } from './migrations/helpers';
+
+/**
+ * This class is responsible for building the transaction used to migrate the liquidity.
+ *
+ * @example
+ * ```typescript
+ * const sdk = new BalancerSDK({
+ *   network: 1,
+ *   rpcUrl: 'https://rpc.ankr.com/eth',
+ * })
+ *
+ * const migrations = new Migrations(
+ *   sdk.networkConfig.addresses.contracts.relayerV4 as string,
+ *   sdk.data.pools,
+ *   sdk.data.liquidityGauges.subgraph,
+ *   sdk.provider
+ * )
+ *
+ * const user = '0xfacec29Ae158B26e234B1a81Db2431F6Bd8F8cE8'
+ * const from = '0x32296969ef14eb0c6d29669c550d4a0449130230000200000000000000000080'
+ * const to = '0x32296969ef14eb0c6d29669c550d4a0449130230000200000000000000000080'
+ * const { to, data } = await migrations.pool2pool(user, from, to)
+ *
+ * const tx = await sdk.provider.sendTransaction({ to, data })
+ * ```
+ */
+export class Migrations {
+  constructor(
```

Comment:
Adding some docs/comments to constructor is useful for the docs popup.

Reply:
Good point